express/properties/material.py

```diff
@@ -60,8 +61,12 @@ def unitCellFormula(self):
     def derived_properties(self):
         derived_properties = []
         try:
-            inchi = Inchi("inchi", self.molecule_parser).serialize_and_validate()
-            inchi_key = InchiKey("inchi_key", self.molecule_parser).serialize_and_validate()
+            if self.is_periodic:
```

Comment:
`inchi = Inchi("inchi", self.molecule_parser).serialize_and_validate() if self.is_non_periodic else None`
same for inchi_key